k8s/aat/common/ccd/definition-store-api.yaml

```diff
@@ -32,7 +32,7 @@ spec:
         DEFINITION_STORE_DB_NAME: ccd_definition_store
         DEFINITION_STORE_DB_HOST: ccd-definition-store-api-postgres-db-v11-aat.postgres.database.azure.com
         DUMMY_RESTART_VAR: 2
-        DEFINITION_STORE_TX_TIMEOUT_DEFAULT: 120
+        DEFINITION_STORE_TX_TIMEOUT_DEFAULT: 40
```

Comment:
@MSancaktutar won't this return frequent 504s on AAT?

Reply:
Anything beyond 30 sec will already receive 504.
TXs beyond 40 sec will be rolled back. Within 30-40 sec will actually take effect. (imports taking 30-40 secs)